spec/01-unit/03-prefix_handler_spec.lua

```diff
@@ -126,10 +126,17 @@ describe("NGINX conf compiler", function()
   describe("compile_nginx_conf()", function()
     it("compiles a main NGINX conf", function()
       local nginx_conf = prefix_handler.compile_nginx_conf(helpers.test_conf)
-      assert.matches("user nobody nobody;", nginx_conf, nil, true)
+      assert.not_matches("user nobody nobody;", nginx_conf, nil, true)
       assert.matches("worker_processes 1;", nginx_conf, nil, true)
       assert.matches("daemon on;", nginx_conf, nil, true)
     end)
+    it("compiles with user directive", function()
```

Comment:
As per the previous comment, the tests could now be grouped like so:

```lua
describe("user directive", function()
  it("is not included by default")
  it("is not included when 'nobody'")
  it("is not included when 'nobody nobody'")
  it("is included when otherwise")
end)
```

I feel like we need that level of granularity and testing for this directive because it will definitely bite us at some point due to all the logic we are adding around it...